index.html

```diff
@@ -79,5 +79,105 @@ <h3>Skills</h3>
       </div>  
   </div>
 </section>
+<section class="work">
+    <div class="worktitle">
+        <h1>MY RECENT<br> WORKS</h1>
+        <hr class="divider">
+    </div>
+    <div class="cards">
+      <div class="card">
+         <div class="profile">
+
+         </div>
+         <div id="firstTitle" class="workText">
+         <h3>Multi-Post Stories</h3>
+         <p>
+            A daily selection of privately personalized reads;
+             no accounts or sign-ups required. 
+             has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a standard dummy text.
+         </p>
+         </div>
+          <ul class="categories">
+             <li>css</li>
+             <li>html</li>
+             <li>Bootstrap</li>
+             <li>Ruby</li>
+          </ul>
```

Comment:
- [ ] Kindly add a separator between technologies as shown in the Figma design.
Figma
<img width="400" alt="Screen Shot 2022-03-18 at 16 33 19" src="https://user-images.githubusercontent.com/28307960/159022697-6177d847-8e1b-4957-b2cf-c295256bc7c8.png">
Your design
<img width="368" alt="Screen Shot 2022-03-18 at 16 33 43" src="https://user-images.githubusercontent.com/28307960/159022743-c221289b-f808-4cd2-927f-042661c293c4.png">

Reply:
Added separator  👍🏽